webapp/src/css/enketo/_fonts.scss

```diff
@@ -1,8 +1,8 @@
 @font-face {
     font-family: 'enketo-icons';
-    src: url('../fonts/enketo-icons-v2.woff') format('woff'),
-        url('../fonts/enketo-icons-v2.ttf') format('truetype'),
-        url('../fonts/enketo-icons-v2.svg#enketo-icons') format('svg');
+    src: url('/fonts/enketo-icons-v2.woff') format('woff'),
+        url('/fonts/enketo-icons-v2.ttf') format('truetype'),
+        url('/fonts/enketo-icons-v2.svg#enketo-icons') format('svg');
```

Comment:
Why are these fonts moving? It seems unrelated to the branding change?

Reply:
This shouldn't be committed, a separate task should be created for this. This came up when I realized that the admin app could not get `NotoSans` font, solution was to create `fonts` folder on the admin app and add `NotoSans-Regular.tff` to it. The only problem with that would be duplication since they already existed on the webapp. So I  decided to serve all the fonts from a central location i.e. `api public` dir

Reply:
If you serve them from there they probably won't be cached for offline use... but we can discuss that in a separate commit!